docker/Dockerfile

```
@@ -16,6 +19,22 @@ ARG VITE_SENTRY_REPLAYS_SESSION_SAMPLE_RATE
 ARG VITE_SENTRY_REPLAYS_ERROR_SAMPLE_RATE
 ARG ENV_API_LOCAL_DIR
 
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends ca-certificates curl libatomic1 xz-utils \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)